Update audit event names in readiness healthcheck RFC (#715)
cloudfoundry/community: toc/rfc/rfc-0020-readiness-healthchecks.md

logs: "Container failed the readiness health check. Container marked not ready
and removed from route pool".

#### App Audit Events

When the liveness healthchecks fail, it results in the following audit events:

> `audit.app.process.crash` and `audit.app.process.rescheduling`.

Reviewer: audit.app.process.rescheduling is sent when an app is terminated because of cell evacuation. I don't think that it is related to the liveness healthcheck at all.

[ccng #2327](https://github.com/cloudfoundry/cloud_controller_ng/issues/2327)
https://docs.cloudfoundry.org/running/managing-cf/audit-events.html#when-are-events-created

Author: Updated the wording. How does it look now?

> When the liveness healthchecks fail, it results in the following audit events:
> `audit.app.process.crash` and `audit.app.process.rescheduling`.

Author: ```suggestion
When a liveness healthcheck fails, it results in the `audit.app.process.crash` audit event.
```

logs: "Container failed the readiness health check. Container marked not ready
and removed from route pool".

> #### App Audit events

Author: ```suggestion
#### App Audit Events
```

When the liveness healthchecks fail, it results in the following audit events:
`audit.app.process.crash` and `audit.app.process.rescheduling`.

> Similarly, when AI readiness healthcheck succeeds a new application event should be emitted:
> `audit.app.process.ready`. And when AI readiness healthcheck fails a new event should be emitted:
> `audit.app.process.notready`.

Author: ```suggestion
Similarly, when an AI readiness healthcheck succeeds an `audit.app.process.ready` event should be emitted.
And when an AI readiness healthcheck fails an `audit.app.process.notready` event should be emitted.
```
